engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -354,8 +355,10 @@ class WorkflowManagerActor(backend: Backend, config: Config)
 
     case class ActorAndStateData(actor: WorkflowActorRef, data: WorkflowManagerData)
 
+    // TODO: Obviously unpleasant, but when workflows don't have backends any more, this goes away. Hooray!
```

Comment:
I second with @scottfrazer there. Just collected them all from intelliJ, there are some 40-odd TODOs (not necessarily all related to PBEs) currently overall, and it worries me how / when we get to clean those up.